fixieai/cli/agent/commands.py

```diff
@@ -223,12 +224,15 @@ def _ensure_agent_updated(
     "--tunnel/--no-tunnel",
     is_flag=True,
     default=True,
-    help="Create a tunnel using localhost.run",
+    help="Create a tunnel using localhost.run.",
+)
+@click.option(
+    "--reload/--no-reload", is_flag=True, default=True, help="Enable auto-reload."
```

Comment:
Can you add "default set" to this flag and the one above please? Right now the help looks a bit confusing and suggests they are not set.

<img width="741" alt="Screen Shot 2023-03-10 at 9 54 53 AM" src="https://user-images.githubusercontent.com/6152401/224388805-7a1eb575-e18a-4fc8-bc40-1dfbbd742a45.png">

Reply:
Good callout! Will do